### ISSUE TITLE
Cannot save after checking hash password

### ISSUE THREAD

**Issue by justinrhodes** [Tautulli/Tautulli#707]
**Version:** 1.4

**Branch:** master

**Commit hash:** a726154b1d2231fdc59d2ffb2d567e2008e68348

**Operating system:** Docker

**Python version:** 2.7.11

**What you did?**
Add HTTP username and password under Access Control with hashed password

**What happened?**
An error occurred after saving when hash password option is checked
PlexPy restarts and logs out
Hash password field is still unchecked and password is not hashed in config.ini

**What you expected?**
Password to be hashed properly in config

**How can we reproduce your issue?**

**What are your (relevant) settings?**

**Link to logs:**

```
2016-05-16 08:16:37 - DEBUG   :: MainThread : Trying to execute: "git rev-parse HEAD" with shell in /opt/plexpy
2016-05-16 08:16:37 - DEBUG   :: MainThread : Git output: a726154b1d2231fdc59d2ffb2d567e2008e68348
2016-05-16 08:16:37 - DEBUG   :: MainThread : Trying to execute: "git rev-parse --abbrev-ref HEAD" with shell in /opt/plexpy
2016-05-16 08:16:37 - DEBUG   :: MainThread : Git output: master
2016-05-16 08:16:37 - INFO    :: MainThread : Retrieving latest version information from GitHub
2016-05-16 08:16:37 - DEBUG   :: MainThread : Requesting URL via GET method: https://api.github.com/repos/drzoidberg33/plexpy/commits/master
2016-05-16 08:16:38 - DEBUG   :: MainThread : Latest version is a726154b1d2231fdc59d2ffb2d567e2008e68348
```

**Comment by Hellowlol** [comment]
The log is completely useless. Full logs plx.

**Comment by justinrhodes** [comment]
I'm not sure where the "full" logs are stored. This is pulled from logs/plexpy.log

These are the lines which occur after trying to save when checking hash password. What other information would you need?

**Comment by Hellowlol** [comment]
Click the download button inside plexpys interface for the logs. We dont know, thats the reason we need it all. Sometime isnt really obvious what the error might be.

**Comment by justinrhodes** [comment]
Cleared logs. Reproduced issue. Downloaded logs...

```
2016-05-16 08:35:50 - INFO    :: CP Server Thread-6 : Cleared the plexpy.log file.
2016-05-16 08:35:56 - INFO    :: MainThread : Checking to see if the database has all tables....
2016-05-16 08:35:56 - DEBUG   :: MainThread : Trying to execute: "git rev-parse HEAD" with shell in /opt/plexpy
2016-05-16 08:35:56 - DEBUG   :: MainThread : Git output: a726154b1d2231fdc59d2ffb2d567e2008e68348
2016-05-16 08:35:56 - DEBUG   :: MainThread : Trying to execute: "git rev-parse --abbrev-ref HEAD" with shell in /opt/plexpy
2016-05-16 08:35:56 - DEBUG   :: MainThread : Git output: master
2016-05-16 08:35:56 - INFO    :: MainThread : Retrieving latest version information from GitHub
2016-05-16 08:35:56 - DEBUG   :: MainThread : Requesting URL via GET method: https://api.github.com/repos/drzoidberg33/plexpy/commits/master
2016-05-16 08:35:57 - DEBUG   :: MainThread : Latest version is a726154b1d2231fdc59d2ffb2d567e2008e68348
2016-05-16 08:35:57 - INFO    :: MainThread : PlexPy is up to date
2016-05-16 08:35:57 - INFO    :: MainThread : PlexPy PlexTV :: Requesting URLs for server...
2016-05-16 08:35:57 - INFO    :: MainThread : PlexPy Config :: Writing configuration to file
2016-05-16 08:35:58 - INFO    :: MainThread : PlexPy Config :: Writing configuration to file
2016-05-16 08:35:58 - INFO    :: MainThread : PlexPy PlexTV :: Using user-defined URL.
2016-05-16 08:35:58 - INFO    :: MainThread : PlexPy Pmsconnect :: Requesting name from server...
2016-05-16 08:35:58 - INFO    :: MainThread : PlexPy PlexTV :: Requesting users list refresh...
2016-05-16 08:35:59 - INFO    :: MainThread : PlexPy PlexTV :: Users list refreshed.
2016-05-16 08:35:59 - INFO    :: MainThread : PlexPy Pmsconnect :: Requesting libraries list refresh...
2016-05-16 08:35:59 - INFO    :: MainThread : PlexPy Pmsconnect :: Libraries list refreshed.
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Check GitHub for updates
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Refresh Plex server URLs
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Refresh Plex server name
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Check for recently added items
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Check for Plex updates
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Check for Plex remote access
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Check for active sessions
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Refresh users list
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Refresh libraries list
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Backup PlexPy database
2016-05-16 08:35:59 - INFO    :: MainThread : Scheduled background task: Backup PlexPy config
2016-05-16 08:35:59 - INFO    :: MainThread : PlexPy WebStart :: Web server authentication is enabled, username is 'JRhodes88'
2016-05-16 08:35:59 - INFO    :: MainThread : PlexPy WebStart :: Starting PlexPy web server on https://0.0.0.0:8181/
2016-05-16 08:36:03 - DEBUG   :: CP Server Thread-4 : PlexPy WebAuth :: Admin user 'JRhodes88' logged into PlexPy.
```

**Comment by JonnyWong16** [comment]
You're not the first to report this [problem](https://forums.plex.tv/discussion/comment/1183635/#Comment_1183635).

I'm not sure what is causing it yet. And those logs also don't show anything. You clicked the "Save" button right? The logs don't say settings were saved so it gives me an idea about where to start looking.

**Comment by justinrhodes** [comment]
Here is a video of what happens. Notice how the error actually logs the user out.
https://vid.me/n2eM

**Comment by JonnyWong16** [comment]
Have you tried a different password?

**Comment by justinrhodes** [comment]
Issue persists with a new password. Confirmed that password is still stored as plain text in config.

Is it an issue with the version of Python perhaps?

**Comment by JonnyWong16** [comment]
I'm also on Python 2.7.11, but it might be some python dependency that's missing.

**Comment by JamminR** [comment]
**Version:** 1.4.0

**Branch**: master

**Commit hash**: (unknown, tell me how to get and I will, I use Zip file download)

**Operating system**: Win7x64

**Python version**: From Plexpy settings page, 
    2.7.11 (v2.7.11:6d1b6a68f775, Dec 5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)]
This also occurred with 2.7.10. Updating to 2.7.11 was part of my troubleshooting.

**What you did?** - After checking "Hash Password in the config file" in PlexPy settings - Access Control page - Clicked "Save"

**What happened?** - python.exe and pythonw.exe crash, presenting Windows "this application has stopped working", submit to find solution or close program options. No hashed password is saved.

**What you expected?** - Plexpy to save hashed password and continue.

**How can we reproduce your issue?** Unknown

**What are your (relevant) settings?** --nolaunch at startup of PlexPy for normal use shortcut with pythonw.exe. As part of troubleshooting, I totally wiped all Plexpy files with exception of the plexpy.db, re-installed/copied from zip to my plexpy folder, restarted plexpy using python command line, answered ONLY the default first run questions, then went to Access Control and attempted checking the box. Once checked, saved, python.exe crashed. 

**Link to logs:** 
[plexpy.txt](https://github.com/drzoidberg33/plexpy/files/269260/plexpy.txt)
(log attached - my login username in the logs was changed to "_deleted_for_privacy_", as I don't use online persona names for my actual login info. - That being said, it is straight ascii alphabetic username, and the password is (currently) 
straight ascii alphanumeric. I removed special symbols in hopes it would help, but it did not.
